src/components/LoanCards/LoanProgressGroup.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<figure>
+		<h4 class="tw-mb-0.5">
+			{{ moneyLeft | numeral('$0,0[.]00') }} to go{{ timeLeft !== '' ? '. ' + timeLeft : '' }}
```

Comment:
Will a template literal work here?
Instead of `'. ' + timeLeft` you might have `. ${timeLeft}` wrapped in backticks (`)...